<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1219</width>
         <height>1218</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>1200</width>
            <height>1200</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Results</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QGroupBox" name="groupBox">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="title">
                  <string>Configuration</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="dataSetLabel_2">
                      <property name="text">
                       <string>Data set :</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="dataSetComboBox">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="6" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Learning rate</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="learningRateInput">
                      <property name="text">
                       <string>0</string>
                      </property>
                      <property name="placeholderText">
                       <string>5000</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="4" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <widget class="QLabel" name="maxIterLabel_4">
                      <property name="text">
                       <string>Amount of classes</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="amountOfClassesInput">
                      <property name="text">
                       <string>0</string>
                      </property>
                      <property name="placeholderText">
                       <string>5000</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="0" column="1">
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLabel" name="learningModelLabel_2">
                      <property name="text">
                       <string>Learning Model :</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="learningModelComboBox">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="1">
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item>
                     <widget class="QLabel" name="maxIterLabel_2">
                      <property name="text">
                       <string>Max Iterations :</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="maxIterInput">
                      <property name="text">
                       <string>0</string>
                      </property>
                      <property name="placeholderText">
                       <string>5000</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="4" column="1">
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <widget class="QLabel" name="maxIterLabel_5">
                      <property name="text">
                       <string>Max classification error</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="maxClassificationErrorInput">
                      <property name="text">
                       <string>0</string>
                      </property>
                      <property name="placeholderText">
                       <string>5000</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="10" column="1">
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>PMC architecutre</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="pmcArchiInput">
                      <property name="text">
                       <string>0</string>
                      </property>
                      <property name="placeholderText">
                       <string>5000</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="7" column="0" colspan="2">
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QRadioButton" name="multiLayerCheckButton">
                      <property name="text">
                       <string>Multilayer</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="monolayerCheckButton">
                      <property name="text">
                       <string>Monolayer</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="11" column="0" colspan="2">
                   <layout class="QGridLayout" name="controlBar">
                    <item row="1" column="1">
                     <widget class="QLabel" name="learningModelStatus">
                      <property name="text">
                       <string>Not ready</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QPushButton" name="startBtn">
                      <property name="text">
                       <string>Start learning</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QPushButton" name="resetBtn">
                      <property name="enabled">
                       <bool>true</bool>
                      </property>
                      <property name="text">
                       <string>Reset</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QLabel" name="maxIterLabel_3">
                      <property name="text">
                       <string>Minimum quadratic mean error</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="minErrorInput">
                      <property name="text">
                       <string>0</string>
                      </property>
                      <property name="placeholderText">
                       <string>5000</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="12" column="0">
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="6" column="1">
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Activation function</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="activationFctComboBox">
                      <item>
                       <property name="text">
                        <string>Linear</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Sigmoïde</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <widget class="QWidget" name="chart" native="true">
               <property name="minimumSize">
                <size>
                 <width>634</width>
                 <height>500</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="1" column="1">
                <widget class="QPushButton" name="previousIterBtn">
                 <property name="text">
                  <string>&lt;</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="firstStepBtn">
                 <property name="text">
                  <string>&lt;&lt;</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="6">
                <widget class="QPushButton" name="lastIterBtn">
                 <property name="text">
                  <string>&gt;&gt;</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="iterationNumLabel">
                 <property name="text">
                  <string>Iteration</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="5">
                <widget class="QPushButton" name="nextIterBtn">
                 <property name="text">
                  <string>&gt;</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QLabel" name="iterationMaxLabel">
                 <property name="text">
                  <string>of 0</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QLineEdit" name="iterationInput">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QListView" name="listView"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1256</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
